src/gql/queries/spawn-task.graphql

```diff
@@ -1,7 +1,7 @@
 #import "../fragments/baseTask.graphql"
 
-query GetSpawnTask($taskId: String!, $execution: Int) {
-  task(taskId: $taskId, execution: $execution) {
+query GetSpawnTask($taskId: String!) {
+  task(taskId: $taskId, execution: 0) {
```

Comment:
Why did we make this change? i think the old behavior was getting the latest task execution while this always gets the first execution.